docs/translations/fr/scripting/functions/AddCharModel.md

```diff
@@ -0,0 +1,59 @@
+---
+title: AddCharModel
+description : Ajoute un nouveau modèle de personnage personnalisé à télécharger.
+tags: []
+---
+
+<VersionWarn version='SA-MP 0.3.DL R1' />
+
+## Description
+
+Ajoute un nouveau modèle de personnage personnalisé à télécharger. Les fichiers modèles seront stockés dans le dossier Documents\GTA San Andreas User Files\SAMP\cache du lecteur sous le dossier IP et port du serveur dans un nom de fichier au format CRC.
+
+| Nom     | Description                                                                                                                                 |
+| ------- | -------------------------------------------------- -----------------------------------------------------------------------------------------|
+| baseid  | L'ID du modèle de skin de base à utiliser (comportement du personnage et personnage d'origine à utiliser lorsque le téléchargement échoue). |
+| newid   | Le nouvel ID de modèle de skin allait de 20 000 à 30 000 (10 000 emplacements) à utiliser ultérieurement avec SetPlayerSkin                 |
+| dffname | Nom du fichier de collision de modèles .dff situé dans le dossier du serveur de modèles par défaut (paramètre artpath).                     |
+| txdname | Nom du fichier de texture de modèle .txd situé dans le dossier du serveur de modèles par défaut (paramètre artpath).                        |
+
+## Retour
+
+1 : La fonction s'est exécutée avec succès.
+
+0 : La fonction n'a pas pu s'exécuter.
+
+## Exemples
+
+```c
+public OnGameModeInit()
+{
+     AddCharModel(305, 20001, "lvpdpc2.dff", "lvpdpc2.txd");
+     AddCharModel(305, 20002, "lapdpd2.dff", "lapdpd2.txd");
+     return 1;
+}
+```
+
+```c
+AddCharModel(305, 20001, "lvpdpc2.dff", "lvpdpc2.txd");
+AddCharModel(305, 20002, "lapdpd2.dff", "lapdpd2.txd");
+```
+
+## Remarques
+
+:::conseil
```

Comment:
this will not work as the Markdown extensions for these admonitions do not translate the actual symbol names in the syntax, you must use English names - it's similar to calling a function in a programming language, generally function names aren't translated but content/data is.